lib/executor/helper.cpp

```diff
@@ -124,6 +124,8 @@ Executor::enterFunction(Runtime::StackManager &StackMgr,
     Span<ValVariant> Args = StackMgr.getTopSpan(ArgsN);
     std::vector<ValVariant> Rets(RetsN);
 
+    SavedThreadLocal SavedThreadLocal;
```

Comment:
Wondering why only save the local stack when entering a AOT compiled function?

Reply:
Currently only `Executor::prepare` overwrites these thread local variables.
So saving them before calling `Executor::prepare`.

Reply:
Yes, the current workflow will only overwrite these thread-local variables inside the AOT-compiled condition before calling `prepare`, which will modify these values.